src/main/java/eu/bbmri/eric/csit/service/negotiator/database/model/Negotiation.java

```diff
@@ -134,6 +144,17 @@ public void setStateForResource(String resourceId, NegotiationResourceState stat
     currentStatePerResource.put(resourceId, state);
   }
 
+  public void addNewRecordToNegotiationResourceStateHistory(
```

Comment:
make the part of setStateForressource.

Reply:
Fixed in 673d2d0e5aa92b06fda147cf34949dd7444d4de4